cdc/kv/etcd.go

```diff
@@ -126,6 +126,11 @@ func NewCDCEtcdClient(ctx context.Context, cli *clientv3.Client) CDCEtcdClient {
 	return CDCEtcdClient{Client: etcd.Wrap(cli, metrics)}
 }
 
+// Close releases resources in CDCEtcdClient
+func (c CDCEtcdClient) Close() error {
```

Comment:
Do we leak the client in non-testing code?

Reply:
I will investigate it

Reply:
But I suggest merging this PR first